tests/cli/test_deploy.py

```diff
@@ -39,6 +39,43 @@ def project_dir(tmp_path):
     os.chdir(original_dir)
 
 
+@pytest.fixture
+def deployment_with_apply(tmp_path):
```

Comment:
This test module tests the `prefect deploy` command (currently in beta) which is different than the `prefect deployment build` and `prefect deployment apply` commands. Can you please move this test to `tests/cli/deployment/test_deployment_build.py`?

Reply:
Thanks!  I removed the test from `test_deploy` and modified and added tests sin `test_deployment_build`.